source/_integrations/sentry.markdown

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
missing: `ha_release`, `ha_category` and the `ha_iot_class`

Reply:
Its unclear to me what ha_category (same w/ ha_iot_class) should be, and given ha_release doesnt yet exist, how do I identify that?

Reply:
"Other" or a new category for it? There's nothing like "Monitoring" or "Logging" afaict

Reply:
also found my answer to ha_iot_class:

ha_iot_class: Cloud Polling

Reply:
Maybe `System Monitor` is appropriate for this integration. For the release version you can use 0.104, prior to a merge we will check if this version is correct and if not we will adjust it.